src/Features/Admin/Vendors/index.js

```diff
@@ -1,28 +1,59 @@
-import React from 'react';
+import React, { useEffect, useState } from 'react';
 
+import { useMutation } from 'react-query';
+import { useSelector } from 'react-redux';
 import { useHistory } from 'react-router';
 
 import CommonButton from '../../../components/Button/Button';
 import CustomTable from '../../../components/CustomTable';
-import { vendorList } from '../../../Mock/VendorList';
 import RouteNames from '../../../routes/RouteNames';
+import { deleteUserById } from '../Common Requests/mutation';
+import { FetchUsers } from '../Common Requests/request';
 import { VendorTitleContainer, VendorTitle } from './style';
 
 function VendorList() {
+  const { token } = useSelector((state) => {
+    const {
+      authReducer: {
+        accessToken: { token },
+      },
+    } = state;
+    return {
+      token,
+    };
+  });
+
+  const [vendors, setVendors] = useState('');
+  const header = ['S.No', 'name', 'email', 'Edit'];
+  const { isLoading, isError, data: vendorsData, error, refetch } = FetchUsers('vendor');
+  const Deletevendor = useMutation(deleteUserById, {
+    onError: () => {},
+    onSuccess: () => {
+      refetch();
```

Comment:
I think if we can alias this refetch as refetchUsers than it will be good and in future if there is any other refetch call than you can also alias it avoid any confusions ..